DISCLAIMER.md

```diff
@@ -0,0 +1,49 @@
+
+Simple Icons provides data on the license under which icons are available. We ask users to carefully consider this when using an icon. Users of Simple icons are expected to request permission to use a brand's icon when necessary. As licenses are subject to change we also ask our users to regularly check if the license of the icons they use have been changed.
+
+The license under which an icon is available can be found in the icon's entry in the JSON data file in the NPM and Packagist packages.
+
+As we rely largely on our community to help us keep Simple Icons up-to-date, the license data for an icon may be outdated. If you find that the license data provided is outdated, please [submit an issue] to [the Simple Icons GitHub repository].
+
+If an icon includes a registered trademark (`®`) or trademark symbol (`™`) the recommendations outlined in [the Simple Icons Contributing Guidelines] are followed to decide whether to include the symbol or not.
+
+Simple Icons cannot be held responsible for any legal activity raised by a brand over the use of their icon through Simple Icons.
+
+## Brand Guidelines
+
+> NOTE: the addition of guidelines to Simple Icons is an ongoing project. Hence, the absence of guidelines for a particular brand does not imply that the brand has no guidelines.
+
+Simple icons provides a link to a brand's _branding guidelines_ (or similar) if the brand provides one. We ask our users to carefully read these guidelines before using the brand's icon. As guidelines are subject to change we also ask our users to regularly check if the brand guidelines of the icons they use have been updated.
+
+The brand guidelines can be found in the icon's entry in the JSON data file in the NPM and Packagist packages, or through the "Brand Guidelines" links on [the Simple Icons website].
+
+As we rely largely on our community to help us keep Simple Icons up-to-date, the guidelines link for a brand may be outdated. If you find that the guidelines are outdated, please [submit an issue] to [the Simple Icons GitHub repository].
+
+## Update of Brands
+
+Should a brand wish for their icon or data to be updated, please [submit an issue] to [the Simple Icons GitHub repository]. In the issue, please explain your affiliation with the company and reasons for the update. We generally release updates once a week, but can occasionally make exceptions for immediate updates of brands.
+
+## Removal of Brands
+
+Should a brand wish for their icon to be removed from the package, please [submit an issue] on [the Simple Icons GitHub repository] explaining your affiliation with the company, and reasons for removal. We generally remove icons that no longer [meet our criteria](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#requesting-an-icon) twice a year in our major releases - but can occasionally make exceptions for immediate removal of brands.
+
+Urgent requests for removal can be made by contacting `removals at simpleicons dot org`.
```

Comment:
I think this should be the first line in this section as, urgent or not, I think both we & the company in question would prefer to deal with such requests privately rather then in a public forum. Also, in the worst case scenario that the request is coming from a legal team, we can't assume they have any familiarity with GH nor will appreciate having to register for an account.

Obviously, the inclusion of this line means that we'll need to hold on the PR until the email hosting is set up.

Reply:
Fair enough. Something along the lines of the following then?

```markdown
Should a brand wish for their icon to be removed from the package, contact `removals at simpleicons dot org`. Alternatively, it is also possible to [submit an issue] ...
```

Reply:
_"explaining your affiliation with the company, and reasons for removal"_ should probably already be mentioned for the email, and then something like "or create an issue with the same information" :thinking:

Reply:
@ericcornelissen just added that in an update.

Reply:
I was responding to the updated version :slightly_smiling_face: in a3f99b4ea686386c08a8411b07b4ae7d960cb78a it says

```
Should a brand wish for their icon to be removed from the package, contact `removals at simpleicons
dot org`. Alternatively, it is also possible to [submit an issue] on [the Simple Icons GitHub repository]
explaining your affiliation with the company, and reasons for removal. [...]
```

My suggestion is something along the lines of 

```
Should a brand wish for their icon to be removed from the package, contact `removals at simpleicons
dot org` explaining your affiliation with the company, and reasons for removal. Alternatively, it is also
possible to [submit an issue] on [the Simple Icons GitHub repository] with the same information. [...]
```

Reply:
Would've helped if I pushed the update before saying I'd done it - wouldn't it? 😅

Reply:
:laughing: I see now, thanks for the update :+1: